src/more-infos/more-info-group.js

```diff
@@ -36,4 +39,40 @@ export default new Polymer({
       ],
     },
   },
+
+  observers: [
+    'statesChanged(stateObj, states)',
+  ],
+
+  statesChanged(stateObj, states) {
+    let groupDomainStateObj = false;
+
+    if (states && states.length > 0) {
+      const baseStateObj = maxBy(states, (s) => Object.keys(s.attributes).length);
```

Comment:
This will make it unpredictable which state gets used. It might not be the best solution but I would prefer the first state just to be predictable.